File: src/HW1/Calculator.cs
namespace HW1
{
    public class Calculator
    {
        public static double Calculate(int arg1, char operation, int arg2)
        {
            var result = operation switch
            {
                '+' => arg1 + arg2,
                '-' => arg1 - arg2,
                '*' => arg1 * arg2,
                ':' => (double)arg1 / arg2,
                '/' => (double)arg1 / arg2,
                _   => 0
            };
            return result;
        }
    }
}

File: src/HW4/FSharpCalculator/Calculator.fs
namespace FSharpCalculator
module Calculator = 
    let calculate (arg1:double) (operation:char) (arg2:double) =
        match operation with
        | '+' -> arg1 + arg2
        | '-' -> arg1 - arg2
        | '*' -> arg1 * arg2
        | '/' -> arg1 / arg2